packages/next/pages/_document.tsx

```diff
@@ -40,6 +40,14 @@ function dedupe(bundles: any[]): any[] {
   return kept
 }
 
+function formatModernPath(path: string) {
```

Comment:
This method doesn't feel right. It's called `formatModernPath` but doesn't do it if the feature is disabled.

IMO, this method should look like this with the remaining code updated:
```js
function formatModernPath(path: string) {
  return path.replace(/\.js$/, '.module.js')
}
```

Reply:
I mainly wanted to move the feature detection into a function because it gets really messy adding it to each statement. Maybe I can call it `formatPath` instead?